semaphorejs/contracts/Ownable.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.4.24;
+pragma solidity ^0.5.0;
```

Comment:
I suggest upgrading to the OZ 2.0 Ownable contract.

I can do this in a separate PR as I'd like to add revert messages too.

https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v2.3.0/contracts/ownership/Ownable.sol

Reply:
Sounds good, let's leave it to a separate PR.